tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1278,18 +1278,22 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiFullMeshTrafficSanity:
 qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolSize:
   skip:
     reason: "Headroom pool size not supported."
+    conditions_logical_operator: OR
     conditions:
       - "https://github.com/sonic-net/sonic-mgmt/issues/12292 and hwsku in ['Force10-S6100'] and topo_type in ['t1-64-lag']"
       - "hwsku not in ['Arista-7060CX-32S-C32', 'Celestica-DX010-C32', 'Arista-7260CX3-D108C8', 'Force10-S6100', 'Arista-7260CX3-Q64', 'Arista-7050CX3-32S-C32', 'Arista-7050CX3-32S-D48C8', 'Arista-7060CX-32S-D48C8'] and asic_type not in ['mellanox']"
       - "asic_type in ['cisco-8000']"
+      - "https://github.com/sonic-net/sonic-mgmt/issues/13503 and platform in ['x86_64-nokia_ixr7250e_36x400g-r0']"
```

Comment:
We can do this for brcm dnx chip  rather than a particular HWSKU.

Reply:
@vmittal-msft ,Updated with  asic_subtype as 'broadcom-dnx'